starlette/applications.py

```diff
@@ -118,9 +118,11 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
         scope["app"] = self
         await self.middleware_stack(scope, receive, send)
 
-    # The following usages are now discouraged in favour of configuration
-    #  during Starlette.__init__(...)
```

Comment:
Ah, I'd forgotten that we *did* already have a comment pointing in this direction.

That's great. Let's keep this comment, but now *also* add the docstrings.

Reply:
I've added that back but the diff is about removing a space which I think is correct now, right?